tests/integration/questionnaires.test.js

```diff
@@ -473,6 +440,8 @@ describe('QUESTIONNAIRES ROUTES - PUT /questionnaires/{_id}', () => {
       });
 
       expect(response.statusCode).toBe(200);
+      const questionnairesCount = await Questionnaire.countDocuments({ _id: questionnairesList[0]._id, ...payload });
+      expect(questionnairesCount).toBe(1);
     });
 
     it('should return 400 if questionnaire status is not PUBLISHED', async () => {
```

Comment:
Le test `'should return 400 if try to update another field'` n'est pas utile non ? C'est le cas pour toutes les routes, un champ non défini dans la route ne peut être passé.

Reply:
done